packages/manager/modules/nutanix/src/dashboard/node-express-order-link-generator.js

```diff
@@ -0,0 +1,31 @@
+import JSURL from 'jsurl';
+import { PRODUCT_ID, DEFAULT_OS_NODE_NUTANIX } from '../constants';
+
+export default class NodeExpressOrderLinkGenerator {
+  constructor(serviceName, nodeTechnicalDetails, quantity = 1) {
+    this.serviceName = serviceName;
+    this.nodeTechnicalDetails = nodeTechnicalDetails;
+    this.quantity = quantity;
+  }
+
+  setQuantity(quantity) {
+    this.quantity = quantity;
+  }
+
+  generateLinkParams() {
+    return JSURL.stringify([
+      {
+        serviceName: this.serviceName,
+        planCode: this.nodeTechnicalDetails.billing.plan.code,
+        quantity: this.quantity,
+        productId: PRODUCT_ID,
+        duration: this.nodeTechnicalDetails.billing.pricing.duration,
+        pricingMode: this.nodeTechnicalDetails.billing.pricing.pricingMode,
+        configuration: [
+          { label: 'dedicated_datacenter', value: 'fra' },
+          { label: 'dedicated_os', value: DEFAULT_OS_NODE_NUTANIX },
```

Comment:
why not using a constant for dedicated_datacenter value like dedicated_os ?

Reply:
Because it's related to this bug : MANAGER-16417

I need replace hardcoded datacenter by a definitive solution. But I not this solution plan for the moment